pkg/model/api/v1/user.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	modelAPI "github.com/perses/perses/pkg/model/api"
+)
+
+type UserSpec struct {
+	FirstName string `json:"firstName,omitempty" yaml:"firstName"`
```

Comment:
`omitempty` missing for yaml, is it normal or is it a trick for something? 👀 
If it's intended maybe explain a bit in a comment 😄